<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>298</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MobiVu</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>268</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSize">
    <property name="title">
     <string>Size</string>
    </property>
    <addaction name="actionFull"/>
    <addaction name="actionHalf"/>
   </widget>
   <widget class="QMenu" name="menuQuality">
    <property name="title">
     <string>Quality</string>
    </property>
    <addaction name="actionLow"/>
    <addaction name="actionMid"/>
    <addaction name="actionHigh"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionFront"/>
    <addaction name="actionRear"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionEcho"/>
    <addaction name="actionM2M"/>
   </widget>
   <addaction name="menuSize"/>
   <addaction name="menuQuality"/>
   <addaction name="menuCamera"/>
   <addaction name="menuMode"/>
  </widget>
  <action name="actionFull">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full</string>
   </property>
  </action>
  <action name="actionHalf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Half</string>
   </property>
  </action>
  <action name="actionLow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
  </action>
  <action name="actionMid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mid</string>
   </property>
  </action>
  <action name="actionHigh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>High</string>
   </property>
  </action>
  <action name="actionFront">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Front</string>
   </property>
  </action>
  <action name="actionRear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rear</string>
   </property>
  </action>
  <action name="actionEcho">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Echo</string>
   </property>
  </action>
  <action name="actionM2M">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>M2M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
